Code/Editor/Settings.cpp

```diff
@@ -1027,7 +1029,8 @@ AzToolsFramework::EditorSettingsAPIRequests::SettingOutcome SEditorSettings::Set
     }
     else
     {
-        return { AZStd::string("Invalid Value Type - supported types: string, bool, int, float") };
+        return AzToolsFramework::EditorSettingsAPIRequests::SettingOutcome{ AZStd::unexpect,
+            AZStd::string("Invalid Value Type - supported types: string, bool, int, float") };
```

Comment:
question: Same here, is it possible to keep `AZStd::unexpect` internal (for now) until all usages move over to `AZStd::expected` without needing to change all callsites?